docs/quickstart.md

```diff
@@ -21,6 +23,22 @@ This will download the `dtm` binary and a `quickstart.yaml` config file to your
 
 > Optional: you can then move `dtm` to a place which is in your PATH. For example: `mv dtm /usr/local/bin/`.
 
+### Install with [asdf](https://asdf-vm.com/) 
+
+```shell
+# Show all installable versions
+asdf list-all dtm
+
+# Install specific version
+asdf install dtm latest
+
+# Set a version globally (on your ~/.tool-versions file)
+asdf global dtm latest
+
+# Now dtm commands are available
+dtm --help
+```
+
```

Comment:
Perhaps it would be more appropriate to create a new document dedicated to how to install dtm.

Reply:
It's a good idea. I will create a new document.But I have not found mkdocs in Makefile. Is there a way testing docs in local? By the way, is there any  suggestions  about `nav`?